### PR TITLE
Serilog 2, Custom Groups, and Event Tags

### PR DESCRIPTION
**Pull request by sirkirby**
* move to latest Serilog v2 package and latest Raygun v5 package
* changed target framework to .net 4.6
* added support for Raygun custom group key
* updated nuspec dependencies
* minor update to comments
* added new tags property name parameter which allows tags to be passed per event, in addition to the global tags set on init
* added usage information to readme

### CODE REVIEW

**Comment by sirkirby** [comment]
@nblumhardt Please let me know if you think this may take a while before you have time to review. We need some of this functionality in a current project, however we are happy to move forward with our own package if necessary. If you would like any changes prior to merge, I should be able to turn those around quickly, just let me know.

Thanks!

File: src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs [hunk 1]
```diff
@@ -29,12 +30,14 @@ namespace Serilog.Sinks.Raygun
     /// </summary>
     public class RaygunSink : ILogEventSink
     {
-        readonly IFormatProvider _formatProvider;
-        readonly string _userNameProperty;
-        readonly string _applicationVersionProperty;
-        readonly IEnumerable<string> _tags;
-        readonly IEnumerable<string> _ignoredFormFieldNames;
-        readonly RaygunClient _client;
+        private readonly IFormatProvider _formatProvider;
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs]: Nit: the Serilog codebase doesn't use explicit `private` or `internal` modifiers.

**Comment by nblumhardt** [comment]
Thanks @sirkirby - this looks good.

I don't use Raygun, nor have a Raygun account set up to test this. I think the changeset and proposed changes look good, but after getting this merged, would you be able to "watch" this repository and help iron out any wrinkles?

I've updated the build version in the CI server to 2.0 so that this will produce a new major version when done.

Only the "private modifier" nitpick there. I will be away from the keyboard today, but should be able to get it merged later tonight if I hear from you by then.

Thanks again for doing this, it's much-needed. Cheers!

**Comment by sirkirby** [comment]
@nblumhardt Just made the modifier changes.

I would be happy to watch for issues and provide fixes if necessary. We are heavy users of Raygun at my current organization and I use it personally as well.

I ran some integration tests on my own account and everything checked out fine, however I will look into build some unit tests for this sink as well.

**Comment by nblumhardt** [comment]
Great, thanks Chris.

**Pull request merged**